src/pages/Register.jsx

```diff
@@ -0,0 +1,281 @@
+                                </span>
+                                <span className={validPwd || !pwd ? "hide" : "invalid"}>
+                                    <FontAwesomeIcon icon={faTimes} />
+                                </span>
+                            </label>
+                            <input
+                                type="password"
+                                id="password"
+                                onChange={(e) => setPwd(e.target.value)}
+                                required
+                                aria-invalid={validPwd ? "false" : "true"}
+                                aria-describedby="instruction"
+                                onFocus={() => setPwdFocus(true)}
+                                onBlur={() => setPwdFocus(false)}
+                            />
+                        </div>
+
+                        <div className="input-group">
+                            <label htmlFor="confirm_pwd">
+                                Confirm Password: 
+                                <span className={validMatch && matchPwd ? "valid" : "hide"}>
+                                    <FontAwesomeIcon icon={faCheck} />
+                                </span>
+                                <span className={validMatch || !matchPwd ? "hide" : "invalid"}>
+                                    <FontAwesomeIcon icon={faTimes} />
+                                </span>
+                            </label>
+                            <input
+                                type="password"
+                                id="confirm_pwd"
+                                onChange={(e) => setMatchPwd(e.target.value)}
+                                required
+                                aria-invalid={validMatch ? "false" : "true"}
+                                aria-describedby="instruction"
+                                onFocus={() => setMatchFocus(true)}
+                                onBlur={() => setMatchFocus(false)}
+                            />
+                        </div>
+
+
+
+                        <button disabled={!validFirstName || !validLastName || !validEmail || !validPwd || !validMatch ? true : false}>Sign Up</button>
+                   
+                        {/* Instructions */}
+                        {getInstruction() && (
+                            <p id="instruction" className="instructions">
+                                <FontAwesomeIcon icon={faInfoCircle} /> {getInstruction()}
+                            </p>
+                        )}
+
+                    </form>
+
+                    <p>
+                        Already registered?<br />
+                        <span className="line">
+                            <a href="/login">Sign In</a>
+                        </span>
+                    </p>
+                </section>
+            )}
+        </div>
+    )
+}
+
+await axios.post(REGISTER_URL, {
```

Comment:
Why are we passing hardcoded values to the API, and not the values from the input fields?

Reply:
It was test data, now removed. I am working on it for the next commit